tests/unit/helpers.js

```diff
@@ -230,6 +230,81 @@ export class DummyRequestLogger {
     }
 }
 
+export class CorsConfigTester {
+    constructor(params) {
+        this._cors = [
+          { allowedMethods: ['PUT', 'POST', 'DELETE'],
+            allowedOrigins: ['http://www.example.com'],
+            allowedHeaders: ['*'],
+            maxAgeSeconds: 3000,
+            exposeHeaders: ['x-amz-server-side-encryption'] },
+          { id: 'testid',
+            allowedMethods: ['GET'],
+            allowedOrigins: ['*'],
+            allowedHeaders: ['*'],
+            maxAgeSeconds: 3000 },
+        ];
+
+        if (params) {
+            Object.keys(params).forEach(key => {
+                this._cors[0][key] = params[key];
+            });
+        }
+    }
+
+    getCors() {
+        return this._cors;
+    }
+
+    constructXml() {
+        const xml = [];
+        xml.push('<CORSConfiguration>');
+        this._cors.forEach(rule => {
+            xml.push('<CORSRule>');
+            ['allowedMethods', 'allowedOrigins', 'allowedHeaders',
+            'exposeHeaders', 'maxAgeSeconds']
+            .forEach(elementArr => {
```

Comment:
Minor: `elementArr` should probably be named something else as it is not an array.